helpers/internal.psm1

```diff
@@ -611,146 +611,130 @@ Function Get-Failures {
 }
 
 
-Function Write-LogFile {

```

Comment:
You failed the build :-) No biggie. This is an easy test change.

![image](https://user-images.githubusercontent.com/12801954/128963984-c1691ccd-98de-45c1-a921-0f57f76440cc.png)

Reply:
fixed in unit tests!